app/templates/views/dashboard/_usage.html

```diff
@@ -1,9 +1,12 @@
 {% from "components/big-number.html" import big_number %}
 
+{% set is_high_volume = sms_cost >= 1_000_000 or letter_cost >= 1_000_000 %}
+{% set big_number_kwargs = {"smaller": not is_high_volume, "smallest": is_high_volume} %}
```

Comment:
```suggestion
{% set has_high_spender = sms_cost >= 1_000_000 or letter_cost >= 1_000_000 %}
{% set big_number_kwargs = {"smaller": not has_high_spend, "smallest": has_high_spend} %}
```

Maybe `spend` instead of `volume`? We talk about volumes elsewhere in terms of raw notifications sent so would be nice to separate these two concepts a bit. Definitely optional though.

Was tempted to go full tongue-in-cheek with `hey_big_spender` but 🤷

Reply:
```suggestion
{% set is_big_spender = sms_cost >= 1_000_000 or letter_cost >= 1_000_000 %}
{% set big_number_kwargs = {"smaller": not is_big_spender, "smallest": is_big_spender} %}
```